operator/src/main/java/oracle/kubernetes/operator/LabelConstants.java

```diff
@@ -21,6 +21,7 @@
   String MODEL_IN_IMAGE_MODEL_SECRETS_HASH = "weblogic.modelInImageModelSecretsHash";
   String MODEL_IN_IMAGE_DOMAINZIP_HASH = "weblogic.modelInImageDomainZipHash";
   String INTROSPECTION_STATE_LABEL = "weblogic.introspectVersion";
+  String MII_UPDATED_RESTART_REQUIRED_LABEL = "weblogic.configurationChangedPendingRestart";
```

Comment:
Please change the label to match the corresponding condition's name: e.g: change label to 'weblogic.configChangesPendingRestart'.

Reply:
done